build.gradle.kts

```diff
@@ -76,6 +76,16 @@ subprojects {
 
   configurations.all {
     exclude("javax.servlet", "servlet-api")
+
+    resolutionStrategy {
+      var okHttpVersion = "4.4.1"
+      force(
+        "com.squareup.okhttp3:okhttp:$okHttpVersion",
+        "com.squareup.okhttp3:okhttp-urlconnection:$okHttpVersion",
+        "com.squareup.okhttp3:okhttp-sse:$okHttpVersion",
+        "com.squareup.okhttp3:mockwebserver:$okHttpVersion",
+        "com.squareup.okhttp3:logging-interceptor:$okHttpVersion")
+    }
```

Comment:
Do you intend to update this in `kork/spinnaker-dependencies`?

Reply:
I don't think this should be propagated to other services yet since it's a potentially breaking change (despite 4.x being an okhttp3 version) for any service calling from kotlin.